src/status.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable comma-dangle, import/prefer-default-export, no-undef */
+import './style.css';
+import getTasks from './display.js';
+
+const list = document.querySelector('ul');
+
+export const checkbox = () => {
+  list.addEventListener(
+    'click',
+    (ev) => {
+      if (ev.target.tagName === 'LI') {
+        const task = ev.target.classList.toggle('checked');
+        task.isCompleted = true;
+        localStorage.setItem('task', JSON.stringify(getTasks));
+      }
+    },
+    false
+  );
```

Comment:
- Give style to your checkbox button.
- Kindly fix this issue that appears in console: 
![Screen Shot 2021-08-18 at 18 32 30](https://user-images.githubusercontent.com/22258274/129985186-123abe30-28f7-4e4c-81ef-94d1a4261ae5.png)

**[OPTIONAL]**

   

-  Add a new JavaScript file and import it as a module:

        -it will contain methods related to drag / drop and sorting functionality.

-     Add event listeners to list items (dragstart, dragover, drop).
-     Update items object's values for index key upon user actions.